db/create-db.sql

```diff
@@ -166,7 +171,7 @@ VALUES
 ('Elen', 'Williams', 'ewillams@gemail.com', 'ewill95'),
 ('Owen', 'Parry', 'oparry@gemail.com', 'mybadpassword');
 
-INSERT INTO user_vehicles
+INSERT INTO user_vehiclesemissions
```

Comment:
I haven't updated this as it would break the link to the table. Did you want to change the table name?